controller/src/beerocks/master/son_master_thread.cpp

```diff
@@ -359,7 +359,6 @@ bool master_thread::handle_cmdu_1905_autoconfiguration_search(Socket *sd,
 bool master_thread::handle_cmdu_1905_autoconfiguration_WSC(Socket *sd,
```

Comment:
and if you don't rename the message name to WSC. I demand that you change the function name here to WIFI_SIMPLE_CONFIGURATION
I think that it will give you enough motivation :)

Reply:
I'll think about it 👍